cli/net.go

```diff
@@ -958,6 +990,10 @@ var NetProtectAdd = &cli.Command{
 		defer closer()
 		ctx := ReqContext(cctx)
 
+		if cctx.NArg() < 1 {
```

Comment:
ditto

Reply:
actually, no, if you follow `decodePeerIDsFromArgs` you'll see that it already errors on no arguments, so let's remove this one